src/main/java/com/scalyr/integrations/kafka/mapping/EventMapper.java

```diff
@@ -57,7 +57,13 @@ public EventMapper(Map<String, String> enrichmentAttrs, List<CustomAppEventMappi
    * to map the Event values from the SinkRecord value.
    */
   public Event createEvent(SinkRecord record) {
-    MessageMapper messageMapper = getMessageMapper(record);
+    MessageMapper messageMapper = getMessageMapper(record).orElse(null);
+    if (messageMapper == null) {
+      if (noEventMapperLogRateLimiter.tryAcquire()) {
+        log.info("No event mapper matches sink record value {}", record.value());
```

Comment:
Should this be warning level?

Reply:
I was going back and forth b/n info and warning when I was writing the code.  Based on your comment and thinking about it more, it does seem `WARNING` would be better.